insights/parsers/ip.py

```diff
@@ -405,6 +404,8 @@ def parse_route(self, parts):
         table_type = None
         if parts[0] in self.IGNORE_TYPES:
             return None
+        if parts[0] == 'local':
```

Comment:
By moving the `local` to `SAVED_TYPES`, this logic won't be changed.

Reply:
Done